pkg/annotation/encryption.go

```diff
@@ -0,0 +1,18 @@
+// Package annotation defines annotation keys used by Giant Swarm within
+// Kubernetes resources.
+package annotation
+
+// EncryptionRotationInProgress indicates that there is key rotation in progress. The annotation is stored on the encryption-provider-config secret.
+EncryptionRotationInProgress = "encryption.giantswarm.io/rotation-in-progress"
+
+// EncryptionForceRotation if annotation is set on the ecrnyption-provider-config secret, it will force the key rotation immediately regardless of the age of current encryption key. The annotation is stored on the encryption-provider-config secret.
```

Comment:
```suggestion
// EncryptionForceRotation if annotation is set on the encryption-provider-config secret, it will force the key rotation immediately regardless of the age of current encryption key. The annotation is stored on the encryption-provider-config secret.
```